common.js

```diff
@@ -1728,6 +1728,586 @@ const allSoloCars = {
       'all': ['ssp'],
     },
   },
+  'Nissan': {
+    '200SX': {
```

Comment:
All the 200SX models should be covered under this one. Make the year range 1976 to 2002 and add the classes to each year as appropriate. Every year should have `hs`, 95-98 should have `sts`, every year should have `dsp`, and 76-88 should have `fsp`, 77-88 should have `dp`, 95-99 should have `ep`. Thanks!

Reply:
Addressed in the following commit:
https://github.com/Bjorn248/scca_classifier/pull/81/commits/a57bfa1154f3a813988b94a5161af6e9868a1c45